internal/controller/accountiam_controller.go

```diff
@@ -127,9 +127,7 @@ func (r *AccountIAMReconciler) verifyPrereq(ctx context.Context, instance *opera
 	if err != nil {
 		return err
 	}
-	if len(og.Items) != 1 {
```

Comment:
Hi @qpdpQ 
Can you elaborate the impact of this?

I think this is used to check the number of OperatorGroup in one namespace. Ideally no matter which install mode is specified, there is only one operatogroup per namespace.

Reply:
Understand, I think this is because `WATCH_NAMESPACE` is `""` in All namespace mode, so it is listing all the operatogroup in the cluster given ListOptions has `Namespace: ""`
```golang
og := &olmapi.OperatorGroupList{}
err := r.Client.List(ctx, og, &client.ListOptions{
     Namespace: "",
})
```

Could you please check this comment, I believe we would remove the check on OperatorGroup, and use alternative way for API check. https://github.com/IBM/ibm-account-iam-operator/pull/4#discussion_r1638721322

Reply:
@qpdpQ  Can remove the edb check from here as well. This is blocking certain installations, so let's remove now. And then add the updated check in a separate PR